default_settings.lua

```diff
@@ -31,6 +31,7 @@ local settings = {
 	delete_item_on_clearobject = true,
 	use_real_entities = true,
 	entity_update_interval = 0,
+	lua_tube_print_behavior = "log",
```

Comment:
Never used because this setting is read directly to allow changing it without restarts

```suggestion
```

  

Alternatively settings table could be upgraded to handle both static and dynamic settings.
This would be done with additional table for dynamic defaults and __index meta method for settings table.
Only problem with this is that there's no dedicated table for settings in pipeworks but instead it all gets directly into main `pipeworks` namespace and because of this I wouldn't really recommend going there on this PR.